live-build/misc/migration-scripts/dx_verify

```diff
@@ -14,17 +14,132 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+export MDS_SNAPNAME="MDS-CLONE-upgradeverify"
 
-#
-# [WIP]
-# The existence of this file is checked by dx_unpack, thus we provide
-# it as an empty file until the needed functionality is added.
-#
+DEBUG=false
+
+MDS_CLONE=domain0/$MDS_SNAPNAME
+UPGRADE_VERIFY_PATH=/opt/delphix/server/lib/exec/upgrade-verify
+UPGRADE_VERIFY_JAR=$UPGRADE_VERIFY_PATH/upgrade-verify.jar
+LOG_DIR=/var/delphix/server/upgrade-verify
+MGMT_FMRI=svc:/system/delphix/mgmt:default
+BINDIR=/opt/delphix/server/bin
+DX_MANAGE_PG=$BINDIR/dx_manage_pg
+
+function usage
+{
+    echo "usage: $(basename $0) -v <version> -o <report file> -f <report file format> -l <report file locale>"
+    echo "  -v <version>: The Delphix version number we're upgrading to"
+    echo "  -o <report file>: The output file verification should generate containing an upgrade verification report"
+    echo "  -f <report file format>: The format of the upgrade verification report to generate"
+    echo "  -l <report file locale>: The locale the upgrade verification report should target"
+    echo "  -d: Disable Consistent MDS ZFS Data Utility feature"
+    exit 1
+}
+
+function cleanup
+{
+    if ! $DEBUG; then
+        cleanup_postgres
```

Comment:
if `$root` is empty, those cleanup functions will not function properly. We should probably not call them unless it is set.